Common/src/main/resources/assets/simplystatus/lang/zh_tw.json

```diff
@@ -0,0 +1,197 @@
+{
+  "simplystatus.name": "SimplyStatus",
+  "simplystatus.key.openConfig": "開啟設定",
+
+  "simplystatus.assets.create": "建立資源",
+  "simplystatus.assets.create.filename": "資源 ID：",
+  "simplystatus.assets.create.invalid_name": "無效的 ID 名稱！",
+  "simplystatus.assets.create.id_not_correct": "這個 ID 已經存在！",
+  "simplystatus.assets.title": "名字",
+  "simplystatus.assets.author": "作者",
+  "simplystatus.assets.description": "如要使用這組資源，請將「資源」設為：%s",
+  "simplystatus.assets.icons": "圖示",
+  "simplystatus.assets.base": "基本",
+  "simplystatus.assets.remove": "移除",
+  "simplystatus.assets.reload": "重新載入",
+
+  "simplystatus.config.client": "用戶端",
+  "simplystatus.config.client.enable_rpc": "啟用 RPC",
```

Comment:
This line is not current as of 2.2.0-alpha.5
```
-  "simplystatus.config.client.enable_rpc": "Enable RPC"
+ "simplystatus.config.client.show_rpc": "Show RPC"
```

Reply:
Okay i will fix it